src/enums/formEnum.js

```diff
@@ -0,0 +1,4 @@
+export const formEnum = Object.freeze({
```

Comment:
Be more specific with your variable names, "formEnum" is too vague, it gives no indication of what the values it holds are used for